Solutions/SeraphicSecurity/SolutionMetadata.json

```diff
@@ -0,0 +1,16 @@
+{
+	"publisherId": "",
```

Comment:
please add the "publisherId". 
below is a of validation fail for metadata. 
![image](https://github.com/Azure/Azure-Sentinel/assets/132428394/9279c851-d252-4945-b662-cb56558e1202)

Reply:
what the publisherId should be ? @v-sudkharat